service/telemetry/config_test.go

```diff
@@ -37,6 +38,18 @@ func TestLoadConfig(t *testing.T) {
 			},
 			success: false,
 		},
+		{
+			name: "valid metric telemetry with metric readers",
+			cfg: &Config{
+				Metrics: MetricsConfig{
+					Level: configtelemetry.LevelBasic,
+					Readers: MeterProviderJsonMetricReaders{
+						"pull/prometheus": PullMetricReader{},
```

Comment:
Can you add a test where we don't have a `/` in the name? What should happen then?

Reply:
I added a test case for this in `TestUnmarshalMetricReaders`